tracking/src/main/java/org/hps/recon/tracking/gbl/GBLOutputDriver.java

```diff
@@ -31,16 +31,20 @@
 import org.lcsim.geometry.Detector;
 import org.lcsim.util.Driver;
 import org.lcsim.util.aida.AIDA;
+import hep.aida.IManagedObject;
+import hep.aida.IBaseHistogram;
+
 
 /**
  * Make post-GBL plots needed for alignment.
  *
  * @author Miriam Diamond <mdiamond@slac.stanford.edu>
+ * @author Alessandra Filippi <filippi@to.infn.it>
  */
 public class GBLOutputDriver extends Driver {
 
-    public AIDA aida;
-    private String outputPlots = "GBLplots.root";
+    private AIDA aidaGBL; // era public 
+    private String outputPlots = "GBLplots_ali.root";
```

Comment:
Why change on the name of the outputPlots from default? It can be changed from the driver anyway. 
And I don't understand why public aida to private aidaGBL. Is there a motivation for changing type and name of the variable?